docs/supported-technologies.asciidoc

```diff
@@ -304,6 +304,38 @@ NOTE: only classes from packages configured in <<config-application-packages>>
 
 |===
 
+[float]
+[[supported-java-methods]]
+=== Java method monitoring
+
+If you are seeing gaps in the span timeline and want to include certain methods, there are several options:
+
+|===
+|Style |Description | Advantages | Disadvantages
+
+|API/Code
+|Get the <<api-current-span, current span>> (possibly created by auto instrumentation) and <<api-span-start-span, create a child>> span.
+ You can also use the <<opentracing-bridge, OpenTracing API>> instead of the Elastic specific API.
+|This is the most flexible way: you can add labels to the span and customize the span type, name and timestamp.
+|Wrong usage of the API may lead to invalid traces (scope leaks).
+
+|Annotations
+|Annotate any method with <<api-capture-span>> or <<api-capture-transaction>>.
+|Easier and more robust as there's nothing you can do wrong like forgetting to end a span or close a scope.
+ It's also a bit more performant than the programmatic way.
+|Less flexible on it's own but can be combined with the API.
+ Just get the <<api-current-span, current span>> on an annotated method and customize the span to your liking.
+
+|Configuration
+|Use the <<config-trace-methods>> configuration option to specify additional methods to instrument.
+|You don't need to modify the source code.
+ That makes it possible to monitor code in 3rd party libraries.
+ You can match  methods via wildcards in the package, class or method name,
+ by their modifier (like `public`), by a particular annotation, and more.
+|Easy to overuse by matching too many methods which hurts performance.
```

Comment:
```suggestion
|Easy to overuse by matching too many methods which hurts performance. Use together with <<config-trace-methods-duration-threshold, `trace_methods_duration_threshold`>> if setting for entire packages instead of specific methods in order to reduce overhead and avoid over-verbosity.
```